lib/interpreters/split_clustering.rb

```diff
@@ -278,10 +278,10 @@ def self.split_solve_core(service_vrp, job = nil, &block)
             }.sum
 
             msg =
-              if message.include?('max split')
+              if message.include?('recursive split process')
                 message
               else
-                add = "max split #{(service_vrp[:split_denominators].last * avc).to_i}/#{service_vrp[:split_denominators].last}"
+                add = "recursive split process #{(service_vrp[:split_denominators].last * avc).to_i}/#{service_vrp[:split_denominators].last}"
```

Comment:
Can we do `split solve process` that way we can keep the split_solve name that we have/know/love :heart: